src/main/java/uk/gov/hmcts/divorce/divorcecase/model/Application.java

```diff
@@ -503,6 +504,11 @@ public boolean isSolicitorServiceMethod() {
         return SOLICITOR_SERVICE.equals(solServiceMethod);
     }
 
+    @JsonIgnore
+    public boolean isPersonalServiceMethod() {
+        return PERSONAL_SERVICE.equals(solServiceMethod);
```

Comment:
I think we need to rename solServiceMethod, to serviceMethod

Reply:
Complete
https://github.com/hmcts/nfdiv-case-api/pull/1671/commits/a2c9da9af8fa17ba82e1aa54f58b17fdd8b18178